### PR TITLE
Css ecommerce page

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  margin: auto;
+  overflow: hidden;
+}
+
+img {
+  width: 100%;
+  margin: 0 -5px 0 -5px;
+
+  /* object-fit: cover;
+  object-position: 20% 10%; */
```
Comment (thebearingedge) [inline css-ecommerce-page/styles.css:50]: Please delete commented out code before turning in your solution.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Owen, just a few notes on your solution.

- The background color should be left as white for this one.
- Try to reorganize your HTML into columns.
- The sale prices should have a subtle background color to them.

**Comment by owen-adams2021** [comment]
Updated Screenshot
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Owen! Rather than grouping the images and text into separate rows, you should have one `row` container with each image and relevant text for that image in its own `column` container. Please reformat the HTML and do any revisions necessary to your CSS as a result of the change, then resubmit.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Owen! The changes look good in the code, but please provide an updated screenshot after the changes you made to make sure everything still looks correct 👍

**Comment by owen-adams2021** [comment]
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**